CourseDevelopment/Week5/Chapter5.MD

```diff
@@ -1,104 +1,181 @@
++ Extra: https://www.youtube.com/watch?v=iF5-aDJOr6U (virtual lab)
++ Answer the question: How does the future of hardware look like in 2030, 2040 and 2050? Answer in 5 keywords. So the student will bring 3 times 5 keywords to the lecture.
++ Answer the question: What role can open hardware take in society?
++ Resource the game: A little game similar to a sudoku, where students have to solve a recourses problem. Must be developed and it must not be too hard! The answer should hint to design devices made from widely available materials.
++ Extra (Measurement failures): https://www.nature.com/articles/nature.2012.10249 
++ Extra (Measurement failures): https://www.rug.nl/about-ug/latest-news/press-information/scientists-in-focus/romeijnneutrinoen.pdf
 
-**Preparation:** Read article about how physics is represented in Dutch newspapers. https://jcom.sissa.it/archive/20/07/JCOM_2007_2021_A02
 
-**Part 1** A short “classic” lecture of 25 min on:\
-Academic social contract\
-Bounds of expertise\
-Naratives and framing\
-Trust in science\
-Difficulties to reach audience\
-Trust in science
+### Lecture format
+The lecture contains 2 parts. Both answer a specific question related to open hardware and its possible role in society. In preperation, the students have already thought about these questions by reading the materials above. 
 
-**Part 2** Guest lecture & Discussion (60 min)\
-On representation of physics in the Netherlands\
-Lecture by e.g. [Julia Cramer (Leiden University)](https://www.universiteitleiden.nl/en/staffmembers/julia-cramer#tab-1), followed up by a plenary discussion.
+**Part 1** How does the future of hardware look like? (30 min) 
 
+At the beginning of the lecture, a word cloud is created with the answers of the students for 2030, 2040 and 2050. Then the lecture starts covering the topics: Scientific design environments, open protocols and human-computer interfaces, Automation and bots, remote labs, prototyping. In the above question will be answered by the teacher, and this will be compared with the student input. 
 
-## Tutorial Week 5: What is the role of a scientist in society?\
-(2x 60 min)
-\
-**Topics in tutorial:**\
-Metaphors | Communicating uncertainty | possible roles of a scientist
 
-### Tutorial format
-+ **Assignment objective**
-Students practice presenting and express their points of view, by reflecting on their role as a future scientist or citizen. They will use metaphors to communicate topics of their bachelor research. 
+**Part 2** What physical and political constrains and advances is open hardware facing? (30min)
+ 
+recourse the game will be showed and the correct answer will be displayed together with a quote of Sanli Faez. “Sanli: The difference between open hardware and open software is that copying code does not require extra materials, while reproducing hardware is material dependent. So one should consider the choice of materials and access to it if real openness is a goal.””
```

Comment:
What is 'recourse the game'? Needs a link?

Reply:
have not worked on this chapter yet!